controllers/volumereplication_controller.go

```diff
@@ -84,9 +85,16 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return reconcile.Result{}, err
 	}
 
+	instance.Status.LastStartTime = metav1.Now()
+	if err = r.Client.Update(context.TODO(), instance); err != nil {
```

Comment:
we don't need to add the `LastStartTime` when the cluster is being deleted.

Reply:
Moved this after deletion logic